dev/README_RELEASE_AIRFLOW_UPGRADE_CHECK.md

```diff
@@ -0,0 +1,642 @@
+Changes in import path of remote task handlers....................................SUCCESS
+Connection.conn_id is not unique..................................................SUCCESS
+Use CustomSQLAInterface instead of SQLAInterface for custom data models...........SUCCESS
+Found 1 problem.
+
+==========================================================================================
+
+Users must set a kubernetes.pod_template_file value
+---------------------------------------------------
+Skipped because this rule applies only to environment using KubernetesExecutor.
+```
+
+You can then perform any other verifications to check that the it works as you expected.
+
+# Publish the final release
+
+## Summarize the voting for the release
+
+Once the vote has been passed, you will need to send a result vote to dev@airflow.apache.org:
+
+Subject:
+
+```
+[RESULT][VOTE] Release apache-airflow-upgrade-check 1.3.0 from 1.3.0rc1
+```
+
+Message:
+
+```
+Hello Airflow Community,
+
+The vote to release Apache Airflow Upgrade Check 1.3.0 based on 1.3.0rc1 is now closed.
+
+The vote PASSED with 3 binding "+1", 3 non-binding "+1" and 0 "-1" votes:
+
+"+1" binding votes received:
+- Kaxil Naik
+- Jarek Potiuk
+- Xiaodong Deng
+
+"+1" non-Binding votes:
+- Dennis Akpenyi
+- Ephraim Anierobi
+- Vikram Koka
+
+Vote thread:
+https://lists.apache.org/thread.html/r76ec33d9a446e0e08d9d036c0e1550338ce47a5bab98ffb19be52980%40%3Cdev.airflow.apache.org%3E
+
+I'll continue with the release process, and the release announcement will follow shortly.
+
+Regards,
+<your name>
+```
+
+## Publish release to SVN
+
+You need to migrate the RC artifacts that passed to this repository:
+https://dist.apache.org/repos/dist/release/airflow/upgrade-check/
+(The migration should include renaming the files so that they no longer have the RC number in their filenames.)
+
+The best way of doing this is to svn cp between the two repos (this avoids having to upload
+the binaries again, and gives a clearer history in the svn commit logs):
+
+```shell script
+# First clone the repo
+export RC=1.3.0rc1
+export VERSION=${RC/rc?/}
+svn checkout https://dist.apache.org/repos/dist/release/airflow airflow-release
+
+# Create new folder for the release
+cd airflow-release/upgrade-check
+svn mkdir ${VERSION}
+cd ${VERSION}
+
+# Move the artifacts to svn folder & commit
+for f in ../../../airflow-dev/upgrade-check/$RC/*; do svn cp $f ${$(basename $f)/rc?/}; done
+svn commit -m "Release Airflow Upgrade Check ${VERSION} from ${RC}"
+
+# Remove old release
+# http://www.apache.org/legal/release-policy.html#when-to-archive
+cd ..
+export PREVIOUS_VERSION=1.2.0
+svn rm ${PREVIOUS_VERSION}
+svn commit -m "Remove old Upgrade Check release: ${PREVIOUS_VERSION}"
+```
+
+Verify that the packages appear in [airflow upgrade check](https://dist.apache.org/repos/dist/release/airflow/upgrade-check/)
+
+## Prepare PyPI "release" packages
+
+At this point we release an official package:
```

Comment:
(Future enhancement: We should really work out how to just release the artifacts we votes on, rather than build new ones)

Reply:
I believe we should simply take the packages we vote on from SVN after we renamed them. I see no reason why it should not work. It should work the same for airflow, upgrade check and providers.